src/Inventory.zig

```diff
@@ -349,11 +364,13 @@ pub const Sync = struct { // MARK: Sync
 			return inventories.items[serverId].inv;
 		}
 
-		pub fn getInventoryFromSource(source: SourceType) ?Inventory {
+		pub fn getUserInventoryFromSource(user: *main.server.User, source: SourceType) ?Inventory {
```

Comment:
Why not put the user into the source? That's what the `Source` is intended to be after all.